src/textual/css/stylesheet.py

```diff
@@ -433,6 +451,40 @@ def apply(
                 for name, specificity_rules in rule_attributes.items()
             },
         )
+
+        # Set initial values
+        for initial_rule_name in initial:
+            # Rules with a value of None should be set to the default value
+            if node_rules[initial_rule_name] is None:  # type: ignore[literal-required]
+                # Exclude non default values
+                # rule[0] is the specificity, rule[0][1] is 0 for default
+                default_rules = [
+                    rule
+                    for rule in rule_attributes[initial_rule_name]
+                    if not rule[0][0]
```

Comment:
The comment talks about `rule[0][1]` but the code uses `rule[0][0]`, which lost me.

Reply:
Fixed the comment